README.md

```diff
@@ -19,32 +19,32 @@ return el.toReact()
 // Yields: <div style='color: red;' className='box'></div>
 ```
 
-It supports a wide range of DOM operations and will fool most libraries but it isn't exhaustive (the full DOM API is ludicrously large). It supports enough to work with D3 but will require you to fork and add to the project if you encounter something that's missing.
+There's also a **mixin** with animation support:
 
-You can think of this as a bare bones [jsdom][] that's built to bridge the gap between the declarative React and the imperative JavaScript world. We just need to expand it as we go along since jsdom is a huge project that solves different problems.
+```javascript
+// inside componentWillMount
+var faux = this.connectFauxDOM('div','chart');
+d3.doingAnAdvancedAnimationFor3secs(faux);
+this.animateFauxDOM(3500); // duration + margin
 
-I'm trying to keep it light so as not to slow down your render function. I want efficient, declarative and stateless code, but I don't want to throw away previous tools to get there.
+// inside render
+return {this.state.chart};
+```
 
-## Limitations
 
-It's great for...
+React-faux-DOM supports a wide range of DOM operations and will fool most libraries but it isn't exhaustive (the full DOM API is ludicrously large). It supports enough to work with D3 but will require you to fork and add to the project if you encounter something that's missing.
```

Comment:
Nitpick: If it's hyphenated keep it all lowercase and usually wrapped in backticks (for a code block). In this context I would write it as "ReactFauxDOM" or just "The faux DOM..."